### DISCUSSION TITLE
http and https url in sitemap.xml

### DISCUSSION THREAD

**Discussion by irfan-ozer** [spatie/laravel-sitemap#354] [Show and tell]
Hi Dear Friend,
I installed your package. But i have some issues,
i want to use https url in my website but now i have http and https url in my sitemap.xml then i want to remove http url. How can i remove this.
thank you

**Comment by lucasweaver10** [comment]
Same thing for me. The only place I see where you can set it is in the handle() function and I only have https there. Anything we're doing wrong?
